Source/ActivityPub.Types/Conversion/JsonLdSerializer.cs

```diff
@@ -76,8 +89,20 @@ ListableReadOnlyConverter listableReadOnlyConverter
     /// <inheritdoc />
     public T? Deserialize<T>(string json) => JsonSerializer.Deserialize<T>(json, SerializerOptions);
 
+    /// <inheritdoc />
+    public T? Deserialize<T>(Stream json) => JsonSerializer.Deserialize<T>(json, SerializerOptions);
+    
+    /// <inheritdoc />
+    public ValueTask<T?> DeserializeAsync<T>(Stream json) => JsonSerializer.DeserializeAsync<T>(json, SerializerOptions);
+
     /// <inheritdoc />
     public object? Deserialize(string json, Type type) => JsonSerializer.Deserialize(json, type, SerializerOptions);
+    
+    /// <inheritdoc />
+    public object? Deserialize(Stream json, Type type) => JsonSerializer.Deserialize(json, type, SerializerOptions);
+
+    /// <inheritdoc />
+    public ValueTask<object?> DeserializeAsync(Stream json, Type type) => JsonSerializer.DeserializeAsync(json, type);
```

Comment:
@jenniferplusplus wait, I missed this - we need to pass in `SerializerOptions`.

Reply:
woops, that's what I get for not writing tests, I guess. I'll fix this when I can.

Reply:
@jenniferplusplus no worries, I can fix it myself. I noticed that this gets us halfway to completing #60, so I might just go ahead and add the other overloads while I'm at it.